packages/broken/broken.0.4.2/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "1.2"
+version: "0.4.2"
+maintainer: "michipili@gmail.com"
+authors: "Michael Grünewald"
+license: "CeCILL-B"
```

Comment:
Thanks; I hadn't seen the FAQ since the v2 release.  It doesn't really make any statement about the BSD license, only about (the already very ambiguously worded) GPL.  I'd just note that the converse effect also holds: I'm very reluctant to use a non-standard licensed project, and the CeCILL very much falls under "niche license", and so I'd be unlikely to adopt this test harness in my own code.  This shouldn't be more than a passing concern to you of course -- you should pick your license of choice according to your immediate needs :)